src/keys-builder/utils/extract-keys.ts

```diff
@@ -16,8 +16,9 @@ export function extractKeys(
 ): ExtractionResult {
   let { scopeToKeys } = initExtraction();
 
+  
   const fileList =
-    files || input.map((path) => glob.sync(`${path}/**/*.${fileType}`)).flat();
+    files || input.map((path) => glob.sync(`${path.replace(/\\/g, "/")}/**/*.${fileType}`)).flat();
```

Comment:
Can't we use `path.resolve`?

Reply:
Yeah you're right I will update the pr and rename the path variable coming from the input.map(path) to `inputPath` to avoid naming colision there